src/features/remoteExecution/RemoteExecutionEv3Types.ts

```diff
@@ -5,6 +5,34 @@ import SensorGyro from 'src/assets/sGyro.svg';
 import SensorTouch from 'src/assets/sTouch.svg';
 import SensorUltrasonic from 'src/assets/sUltrasonic.svg';
 
+// (18 March 2022)
```

Comment:
Does this get resolved with SWC? I assume not, that's why you've added it.

Might need to take a deeper look into this.

Reply:
Yep it didn't get resolved. SWC is only to parse TS/X files, so it won't do anything to these other imports